modules/matrix_controller.py

```
@@ -19,329 +17,477 @@
-                        }
-                        self.send_json_response(status)
-
-                    elif path == "/config":
-                        config_data = {
-                            "connectionMode": controller.config.get(
-                                "connection_mode", "USB"
-                            ),
-                            "serialPort": controller.config.get("serial_port", ""),
-                            "baudRate": controller.config.get("baud_rate", 115200),
+        class WebHandler(BaseHTTPRequestHandler):
+            def send_cors_headers(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `send_cors_headers`

Add return type annotation: `None`

(ANN202)